Probability_of_Superiority

```diff
@@ -0,0 +1,39 @@
+psup <- function(d=NULL, CL=NULL, plot=F, AUC=F) {
```

Comment:
okomentujes mi semka plz, co by mely tyhle argumenty byt (d, CL)? At to pak muzu nejak srozumitelne napsat do dokumentace :)

Reply:
'd' chapu, ale 'CL' moc ne :)

Reply:
Uf, dalo mi to zabrat, ale je to jasné. Jde o common language effect size z podle [Ruscio (2008)](https://pdfs.semanticscholar.org/34b2/94fa2883a3af79fcb2f40f1e7c8288546fcf.pdf). Každopádně d je Cohenovo d, které převádíš na CL, a nebo naopak.